service_implementation/advanced_plm.md

```diff
@@ -1 +1,456 @@
-# Beyond just files: Using PLM for versioning and metadata
\ No newline at end of file
+# Beyond just files: Using PLM for versioning and metadata
+This document describes how to use PLM services from _inside_ a service, i.e., by calling the PLM API. The text includes  example code.
+There are two types of PLM services for use in CAxMan; both are described below:
+1) those that are integrated in the GSS framework;
+2) the PLM Server native web-services.
+
+## PLM API through GSS
+
+The PLM Server has its own implementation of the GSS services, that is, PLM service endpoints can be obtained through GSS. GSS offers SOAP and REST services:
+- WSDL (SOAP): https://caxman.clesgo.net/jotne/GSS/FileWebService?wsdl
+- REST end point: https://caxman.clesgo.net/jotne/GSS/resources/plm
+
+GSS covers a small subset of simplified methods (services) of the PLM Server. The remaining PLM Server functionality is only available as native PLM Server SOAP web-services; see below for details.

```

Comment:
Suggested change:
Some of the PLM functionality is available as an _extension to the GSS service_ in the sense that the implementation is similarly structured as the GSS service and accepts the same GSS-specific arguments. The remaining ...

Reply:
done